File: dev/requirements/screen/cases/countdown-neutral.2.5.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.5 — the countdown with eleven minutes still in hand — bone, not shouting.
let countdownNeutral_2_5 = RequirementCase(id: "2.5") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "countdown-neutral", id: "2.5", region: .clocks)
    }
}
#endif

File: dev/requirements/screen/cases/countdown-neutral.2.5.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.5 — With time in hand the countdown is bone-coloured.
final countdownNeutral_2_5 = RequirementCase('2.5', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.clocks,
    slug: 'countdown-neutral',
    id: '2.5',
  );
});
